docs/buildah.md

```diff
@@ -22,6 +22,10 @@ The buildah package provides a command line tool which can be used to:
 
 Print debugging information
 
+**--default-mounts-file**
+
+path to default mounts file (default path: "usr/share/containers/mounts.conf")
```

Comment:
The path in the help message is missing the initial `/` that the definition in `secrets.go` includes.